shenfun_elasticity/solver/elastic_problem.py

```diff
@@ -86,6 +171,58 @@ def postprocess(self):
         self.write_xdmf_file(output)
         os.chdir('../../..')
 
+    def _process_boundary_conditions(self):
+        assert hasattr(self, "_bcs")
+        if self._dim == 2:
+            for boundary, bc in self._bcs.items():
+                assert boundary in ('right', 'top', 'left', 'bottom')
+                assert isinstance(bc, list)
+            map_boundary_to_direction_index = {'right': 0, 'top': 1, 'left': 0,
+                                               'bottom': 1}
+            map_boundary_to_side = {'right': 'right', 'top': 'right',
+                                    'left': 'left', 'bottom': 'left'}
+        # base dictionary for boundary conditions in shenfun style
+        bc_base = {'left': list(), 'right': list()}
+        # repeat base dictionary for every component and direction
+        self._processed_bcs = [[deepcopy(bc_base) for _ in range(self._dim)]
+                               for _ in range(self._dim)]
```

Comment:
Instead of `deepcopy(bc_base)` you could have written `{'left': list(), 'right': list()}`...